cookbooks/stateofthemap/recipes/jekyll.rb

```diff
@@ -56,15 +57,35 @@
     group "nogroup"
   end
 
+  archive_file "/srv/#{year}.stateofthemap.org/vendor/bundle-tar-cache" do
+    action :nothing
+    destination "/srv/#{year}.stateofthemap.org/vendor"
+    path "#{Chef::Config[:file_cache_path]}/stateofthemap-gem-bundle-cache.tar"
+    overwrite false
+    only_if { ::File.exist?("#{Chef::Config[:file_cache_path]}/stateofthemap-gem-bundle-cache.tar") }
+    not_if { ::Dir.exist?("/srv/#{year}.stateofthemap.org/vendor/bundle") }
+    only_if { kitchen? }
```

Comment:
Why limit it to when running under kitchen? Either it is safe, in which case we can always use it, or it isn't.

Mind if you are caching binaries there would then be questions about invalidating the cache when the dependencies of those binaries change.